cluster/gce/windows/README-GCE-Windows-kube-up.md

```diff
@@ -62,6 +62,7 @@ plane only runs on Linux.
 export NUM_NODES=2  # number of Linux nodes
 export NUM_WINDOWS_NODES=2
 export KUBE_GCE_ENABLE_IP_ALIASES=true
+export LOGGING_STACKDRIVER_RESOURCE_TYPES=new
```

Comment:
Should we set this for our test jobs too?

(I recognize it's a different repo so it wouldn't be in this PR.)

Reply:
Yep. We should set it our test configurations.